packages/builder/src/components/integration/QueryBindingBuilder.svelte

```diff
@@ -72,7 +72,7 @@
             bindableOptions,
             customParams?.[binding.name]
           )}
-          {bindableOptions}
+          bind:bindings={bindableOptions}
```

Comment:
We don't need the `bind:` here, it's just passing them in one way so we can remove that.

Reply:
The `bindings` in this case are different to the `bindableOptions` - the `bindings` are the Query bindings that need to have values provided for - not really sure what this can be renamed to.

Reply:
Oh right yea, the generic naming caught me out. That's probably why `bindableProperties` didn't get refactored to `bindings` in this file in the first place.